package.json

```diff
@@ -84,9 +84,6 @@
       ]
     },
     "appId": "com.deltachat.desktop",
-    "directories": {
-      "buildResources": "src"
-    },
```

Comment:
This hack is not needed anymore since we build to `es5/` folder.

Reply:
woot